backend/src/libs/packages/database/base-database.package.ts

```diff
@@ -56,6 +56,9 @@ class BaseDatabase implements Database {
         directory: 'src/db/migrations',
         tableName: DatabaseTableName.MIGRATIONS,
       },
+      seeds: {
+        directory: 'src/db/seeds',
+      },
```

Comment:
```suggestion
```

Reply:
I will have an error without it, I added  it because I wasn't able to use without it. You want me to write the path in the script itself?
![seed_1](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/d9239f62-c2e2-4532-b758-1ebf214235dd)